app/src/main/java/org/lineageos/selfie/MainActivity.kt

```diff
@@ -567,9 +599,27 @@ class MainActivity : AppCompatActivity() {
                 // Update duration text and visibility state
                 videoDuration.text = TimeUtils.convertNanosToString(duration)
                 videoDuration.isVisible = enabled
+
+                // Update video recording pause/resume button visibility state
+                if (duration == 0L) {
```

Comment:
Doing it here just so that we show videoRecordingStateButton at the same time as videoDuration "button"